emp/emp_user_cftemplate.yaml

```diff
@@ -335,8 +335,8 @@ Resources:
               StringEquals:
                 aws:RequestTag/emp.pf9.io: owned
               StringLike:
-                aws:RequestTag/emp.pf9.io/baremetalpool: '*'
-                aws:RequestTag/emp.pf9.io/namespace: '*'
+                aws:RequestTag/emp.pf9.io/ns/org-*/bmtpool/*: owned
```

Comment:
nit: can we use `bmtpool` instead of `bmpool`

Reply:
We'll have to change the controllers again and retest